python/paddle/fluid/layers/control_flow.py

```diff
@@ -1729,6 +1730,69 @@ def complete(self):
             })
 
 
+def copy_scope_var_to_global(var, layer_helper):
+    if var is None:
+        return None
+    global_var = layer_helper.create_global_variable(
```

Comment:
How about create variable inside the outer block instead of global block? Think about nested control flow ops.

Reply:
Good catch! Thank you a lot for finding it which can cause buy in nested control flow. It's me who didn't consider well :-)

Reply:
Done.